### PR TITLE
Feature/raz

### PR DESCRIPTION
**Pull request by nmarian85**
Somehow the DCO signed commits are giving me headaches. Retrying.

### CODE REVIEW

**Comment by nmarian85** [comment]
Wow, it worked :). Took me a moment to realize that the Author name and Signed-off-by field need to be the same. Github should document that feature better.

File: plugins/modules/datalake.py [hunk 1]
```diff
@@ -133,6 +133,12 @@
     default: 3600
     aliases:
       - polling_timeout
+  raz:
+    description:
+      - Flag indicating if Ranger RAZ fine grained access should be enabled for the datalake
```
Comment (wmudge) [inline plugins/modules/datalake.py:138]: Also should add entry for the response, which now includes `enableRangerRaz`

File: plugins/modules/datalake.py [hunk 2]
```diff
@@ -537,6 +544,9 @@ def _configure_payload(self):
         if self.scale:
             payload.update(scale=self.scale)
 
+        if self.raz:
+            payload.update(enableRangerRaz=self.raz)
```
Comment (wmudge) [inline plugins/modules/datalake.py:548]: Not supported yet in GCP last time I checked, so will need a guard condition

**Review by wmudge — CHANGES_REQUESTED** [review]
Need to add a guard condition for GCP deployments and update the general response fields to include the RAZ entry

**Comment by wmudge** [comment]
Handled by #55

**Pull request closed without merging**